src/olympia/pages/templates/pages/shield_study_7.html

```diff
@@ -40,31 +40,24 @@ <h3 id="all-shield-studies">All Shield Studies</h3>
 <h3 id="this-particular-study">This Particular Study</h3>
 <p>In addition to the data collected by all Shield Studies and your voluntary answers to the followup survey, here are the <strong>key things you should know</strong> about what is happening when you participate in this study:</p>
 <ul>
-<li>A <em>Session</em> ping will be sent to Mozilla servers when you leave the about:newtab.</li>
-<li>An <em>Event</em> ping will be sent to Mozilla servers when you interact with the about:newtab page (Top Sites, Highlights, Search).</li>
-<li>A <em>Performance</em> ping will be sent to Mozilla servers occasionally to help track responsiveness and latency.</li>
+<li>We collect basic usage data regarding how you interact with the experimental new tab and timeline pages, such as the types of elements that you click and their position on the page. </li>
+<li>We also collect basic information about your browser profile that affect the study, such as the number of bookmarks and history items.</li>
+<li>To provide this new experience, we fetch metadata about websites you have visited, such as images and icons, using a service called <a href="http://embed.ly/">Embedly</a>. We send these requests to Embedly through Mozilla’s servers, so they will not be associated with you.</li>
 <li>As you would expect, we do not collect any information when you are in private browsing mode.</li>
 </ul>
 <p>You can learn more about the data collection for this study <a href="https://github.com/mozilla/activity-stream/blob/master/data_events.md">here</a></p>
```

Comment:
To improve the accessibility of this link could it be wrapped around more than just the word "here" to provide more context?